packages/hardhat-core/sample-projects/typescript/test/Lock.ts

```diff
@@ -4,24 +4,27 @@ import {
 } from "@nomicfoundation/hardhat-toolbox/network-helpers";
 import { anyValue } from "@nomicfoundation/hardhat-chai-matchers/withArgs";
 import { expect } from "chai";
-import { ethers } from "hardhat";
+import { ethers, ignition } from "hardhat";
+
+import LockModule from "../ignition/modules/Lock";
 
 describe("Lock", function () {
   // We define a fixture to reuse the same setup in every test.
   // We use loadFixture to run this setup once, snapshot that state,
   // and reset Hardhat Network to that snapshot in every test.
   async function deployOneYearLockFixture() {
     const ONE_YEAR_IN_SECS = 365 * 24 * 60 * 60;
-    const ONE_GWEI = 1_000_000_000;
+    const ONE_GWEI = 1_000_000_000n;
 
     const lockedAmount = ONE_GWEI;
     const unlockTime = (await time.latest()) + ONE_YEAR_IN_SECS;
 
     // Contracts are deployed using the first signer/account by default
     const [owner, otherAccount] = await ethers.getSigners();
 
-    const Lock = await ethers.getContractFactory("Lock");
-    const lock = await Lock.deploy(unlockTime, { value: lockedAmount });
+    const { lock } = await ignition.deploy(LockModule, {
```

Comment:
I'm not convinced we should be telling people to use Ignition for every test. In this case, we aren't really testing the deployment, so it's not clear that Ignition is adding value.

(same for the rest of the samples)

@fvictorio WDYT?

Reply:
I have reverted the changes to the test files. We have opted not to force Ignition usage where the additional complexity doesn't justify it.